static-site/FAQ.md

```diff
@@ -192,30 +190,38 @@ There is a detailed Confluence page that contains a more in-depth overview of th
 
 60 days
 
-#### When a claim is updated, will a new claim with a different claim ID be created and the previous claim marked as cancelled? OR will the existing claim be updated and the status remain unchanged?
+#### When a claim is updated, will a new claim with a different claim ID be created and the previous claim marked as cancelled? Or will the existing claim be updated and the status remain unchanged?
 
 It's more likely the claim will get updated and the status remains the same. The status will only change to cancelled if the final action field we receive from CCW changes.
 
 ### FHIR Mapping
 
 #### Does the BFD FHIR API send Part A claims data for care received in Critical Access Hospitals and teaching hospitals for Part C enrollees?
 
-No. This information is not provided as part of the CCW-based BFD FHIR API.
+No. This information is not provided as part of the CCW-based BFD FHIR API. While we do have Part C enrollee data, our APIs focus is on Fee for Service Medicare (A, B, & D) and not Medicare Advantage (Part C).
```

Comment:
Spelling out the word feels more readable to me - maybe just personal preference though.
```suggestion
No. This information is not provided as part of the CCW-based BFD FHIR API. While we do have Part C enrollee data, our APIs focus is on Fee for Service Medicare (A, B, and D) and not Medicare Advantage (Part C).
```